fcl/caf/cafmaker_defs.fcl

```diff
@@ -193,6 +193,18 @@ caf_ana_sequence: [ mcreco,
     # TODO: rns??
     ]
 
+caf_ana_data_sequence: [
```

Comment:
I think @gputnam is possibly the originator of this naming scheme, but I think the name is probably because these are some of the additional modules that are run in the CAFMaker fcl but _before_ the `cafmaker` process itself. That said I see the potential raised here for ambiguity between `caf_ana_sequence` and `CAFAna`.

Gray should comment if my reasoning for the title is incorrect or if he is not the originator of the name schema.

Reply:
Yeah that's correct @brucehoward-physics. I don't have strong opinions about the name so feel free to change it to whatever.